android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -217,6 +228,36 @@ private fun SurveyQuestionsContent(questions: List<QuestionContentUiModel>) {
     }
 }
 
+@Composable
+private fun Answer(type: QuestionDisplayType, onAnswerClick: (Int) -> Unit) {
+    when (type) {
+        QuestionDisplayType.STAR -> EmojiRatingAnswer(
+            emojis = MutableList(5) {
+                { Emoji(name = EMOJI_STAR) }
+            },
+            onIndexChange = onAnswerClick
+        )
+        QuestionDisplayType.HEART -> EmojiRatingAnswer(
+            emojis = MutableList(5) {
+                { Emoji(name = EMOJI_HEART) }
+            },
+            onIndexChange = onAnswerClick
+        )
+        QuestionDisplayType.SMILEY -> EmojiRatingAnswer(
+            emojis = listOf(
+                { Emoji(name = EMOJI_POUTING_FACE) },
+                { Emoji(name = EMOJI_CONFUSED_FACE) },
+                { Emoji(name = EMOJI_NEUTRAL_FACE) },
+                { Emoji(name = EMOJI_SLIGHTLY_SMILING_FACE) },
+                { Emoji(name = EMOJI_GRINNING_FACE_WITH_SMILING_EYES) }
+            ),
```

Comment:
```suggestion
            emojis = listOf(
                EMOJI_POUTING_FACE,
                EMOJI_CONFUSED_FACE,
                EMOJI_NEUTRAL_FACE,
                EMOJI_SLIGHTLY_SMILING_FACE,
                EMOJI_GRINNING_FACE_WITH_SMILING_EYES
            ).map { { Emoji(name = it) } },
```

Reply:
@hoangnguyen92dn true, with a very long list, it will have a slight performance issue. However, it's just a 5 items list, for me, I think we can make a trade-off here. It's up to you @suho 😄

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/116/commits/5823aa07c1c30a1dc9437a8f40ad7c587c2f3997